radialgraph/src/main/java/com/duartbreedt/radialgraph/model/Section.kt

```diff
@@ -9,29 +9,34 @@ class Section {
     val value: BigDecimal
     @ColorInt
     val color: Int
+    @ColorInt
+    val gradientColors: List<Int>?
     lateinit var totalValue: BigDecimal
     lateinit var normalizedValue: BigDecimal
     lateinit var percent: BigDecimal
 
-    constructor(value: BigDecimal, color: Int) {
+    constructor(value: BigDecimal, color: Int, gradientColors: List<Int>? = null) {
```

Comment:
```suggestion
    constructor(value: BigDecimal, vararg color: Int) {
```

Reply:
Just checking,
If we make this a list, then do we need to check at some point to make sure that it's not empty or should we let it throw an exception when accessing `first` or `last`?